### ISSUE TITLE
Support for connections 

### ISSUE THREAD

**Issue by fredrike** [ovelius/SlRealtidwidget-ng#4]
I need to see which connections a subway have, would it be possible to limit so only departures with a reasonable connection is visible?

Really love the widget, keep up the good work!

**Comment by ovelius** [comment]
Yes that is reasonable, in fact the code to calculate this is already there: 

// An estimation of how we need to act to travel with this // Departure. enum ProximityAssessment { UNKNOWN = 0; // So close we don't care about time and distance. // This is set because of proximity, not time. RELAXED = 1; // User should start moving towards the stop. MOVE_YOUR_ASS = 2; // User can still make if hurrying towards stop. RUN_FOR_IT = 3; // User can stay at home a while longer and still make it. STAY_PUT = 4; // We are confident the user can't make it. IMPOSSIBLE = 5; }

**Comment by ovelius** [comment]
So, like a checkbox somewhere to enable the filtering, or maybe it should just be default... Hm..
